app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
+
+    like = Like.new(author_id: user.id, post_id: post.id)
```

Comment:
- [ ] <b>REQUIRED:</b> The same thing goes here. Kindly use `current_user` to comply with project requirements.

```ruby
    @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]
```

Always, remember to pass the parameters securely

```ruby
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```